config/config.go

```diff
@@ -822,6 +824,7 @@ var defaultConf = Config{
 		StatusHost:            DefStatusHost,
 		StatusPort:            DefStatusPort,
 		MetricsInterval:       15,
+		MetricsSimplified:     false,
```

Comment:
Please change this configuration to a global-scoped system variable, which is recommended now.

Reply:
Updated. PTAL